Samples/Samples/View/CalendarEventPage.xaml

```diff
@@ -9,58 +9,63 @@
              x:DataType="essentials:Event"
              Title="Event"
              Padding="20,20,20,20">
+    <views:BasePage.Resources>
+        <ResourceDictionary>
+            <x:String x:Key="DateDisplayFormatter">{0:dd/MM/yy hh:mm tt}</x:String>
+        </ResourceDictionary>
+    </views:BasePage.Resources>
     <views:BasePage.Content>
-      <StackLayout>
-          <Label Text="{Binding Title}"
+        <StackLayout>
+            <Label Text="{Binding Title}"
                  FontSize="34"
                  TextColor="Black"
                  HorizontalTextAlignment="Center"/>
-          <Grid>
-              <Grid.ColumnDefinitions>
-                  <ColumnDefinition/>
-                  <ColumnDefinition/>
-                  <ColumnDefinition Width="Auto"/>
-                  <ColumnDefinition/>
-              </Grid.ColumnDefinitions>
-              <Grid.RowDefinitions>
-                  <RowDefinition Height="Auto"/>
-                  <RowDefinition Height="Auto"/>
-                  <RowDefinition Height="Auto"/>
-                  <RowDefinition/>
-              </Grid.RowDefinitions>
-              <Label Grid.Row="1"
+            <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition/>
+                    <ColumnDefinition/>
+                    <ColumnDefinition Width="Auto"/>
+                    <ColumnDefinition/>
+                </Grid.ColumnDefinitions>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto"/>
+                    <RowDefinition Height="Auto"/>
+                    <RowDefinition Height="Auto"/>
+                    <RowDefinition/>
+                </Grid.RowDefinitions>
+                <Label Grid.Row="1"
                      Grid.Column="0"
                      Text="Location:"/>
-              <Label Grid.Row="1"
+                <Label Grid.Row="1"
                      Grid.Column="1"
                      Grid.ColumnSpan="3"
                      Text="{Binding Location}"/>
-              <Label Grid.Row="2"
+                <Label Grid.Row="2"
                      Grid.Column="0"
                      Text="When:"/>
-              <Label Grid.Column="1"
+                <Label Grid.Column="1"
                      Grid.Row="2"
-                     Text="{Binding StartDate, StringFormat='{0:dd/MM/yy hh:mm tt}'}"
+                     Text="{Binding StartDate, StringFormat={StaticResource DateDisplayFormatter}}"
```

Comment:
I like to be explicit about the Binding mode that we're using (e.g. `Mode=OneWay`). This is just my preference though, if you like it without that's perfectly fine - I thought I will mention that.

Reply:
Has been altered accordingly